### ISSUE TITLE
Symbol or label to signify a system app

### ISSUE THREAD

**Issue by teamcons** [elementary/applications-menu#613]
### Problem

Application menu displays all apps present, either from Flatpak, APT, or any .desktop file for, steam apps, 

However flatpak apps offer to deinstall while the others just have a greyed out button. The application menu is aware of what is flatpak and what isnt - but users may be confused why there are apps they cannot uninstall

by default there are the feedback, switchboard, appcenter, multitasking, terminal, shortcuts apps (and later "Monitor")
I can imagine that some users will want to install the official DEB package offered by some apps on their website (here using a minecraft launcher who provides a DEB, but iirc Steam and Discord offer DEB too), so while this format is not encouraged, less savvy users may be confused why they cannot deinstall later

a greyed out label feels like the app cannot be uninstalled for unintelligible reasons.

### Proposal

A way to convey to the user the application cannot be uninstalled by this mean.

Maybe an emblem on bottom left of the icon to signify "this is a system-installed app", and replace the greyed out "Deinstall" label text for these with a greyed out "System app" label to signify "there is no deinstall suggestion because this is embedded in the system now"

**Comment by teamcons** [comment]
![Right cleck](https://github.com/user-attachments/assets/3c596cf8-19eb-4056-9c69-a2054587a72c)

Hmmm

**Comment by teamcons** [comment]
Hey ! Yes. Following our discussion on Discord i meant to come back here and amend the screenshots

The gist of the idea was not other installation methods, but communicating to the user why something cannot be deinstalled or shown in appcenter

Somehow it seems you were left in that discussion on the feeling this is about installation method. It isnt. It's about conveying more info than "no :|"

Not displaying any menu entry sounds worse than greying it out tbh

**Comment by teamcons** [comment]
VS

![Right cleck(4)](https://github.com/user-attachments/assets/0cc71b59-4901-4cc4-b004-e8b204065486)

**Comment by danirabbit** [comment]
> Somehow it seems you were left in that discussion on the feeling this is about installation method. It isnt

This is the title of your issue report :) 

The version of Mail that's installed is actually not the Flatpak version from AppCenter. That's why the menu entries are disabled. If the entry is disabled that means appcenter said "no this is not from me". You would have the same result if you asked Flatpak directly, it would say "no this isn't from me"

**Comment by teamcons** [comment]
> > Somehow it seems you were left in that discussion on the feeling this is about installation method. It isnt
> 
> This is the title of your issue report :)

Yeah just... Lets forget about deb and appimage. The crux of it was more "flatpak" and "not flatpak"

> The version of Mail that's installed is actually not the Flatpak version from AppCenter. That's why the menu entries are disabled. If the entry is disabled that means appcenter said "no this is not from me". You would have the same result if you asked Flatpak directly, it would say "no this isn't from me"

they are enabled for me, but Mail do not appear in Flatpak List.
So appcenter says junk. Ive seen reported "cannot uninstall mail" somewhere so not only for me

Somewhere else you remarked application-menu could solve that itself because of another bug where appcenter didnt say anything, so i was running on that idea: how do we know we can uninstall something or not ?

![Bildschirmfoto von 2024-12-02 20 10 34](https://github.com/user-attachments/assets/3abfbe7d-7384-4379-a2a2-ed71b0852786)

**Comment by danirabbit** [comment]
Oh hm yeah that is weird that mail shows it enabled. I guess because both deb and flatpak versions exist and the applications menu can't know which one this is. That's definitely a bug but it's a different bug I think because that's a false positive
